### PR TITLE
18Ireland: Add stock market and corporations

### PR DESCRIPTION
**Pull request by jenf**
(also attempting to see how splitting config would work)

### CODE REVIEW

**Comment by underhilllabs** [comment]
I like this new format! I think game.rb will be much cleaner, pulling the entity config out into it's own module.

File: lib/engine/game/g_18_ireland/misc_config.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Engine
```
Comment (tobymao) [inline lib/engine/game/g_18_ireland/misc_config.rb:3]: do we really need two configs?

**Review by tobymao — APPROVED** [review]

File: lib/engine/game/g_18_ireland/misc_config.rb (hunk 1 above)
Comment (jenf) [reply lib/engine/game/g_18_ireland/misc_config.rb:3]: it's probably worth the discussion with others, personally i like things split into categories so i can quickly find them. But having multiple files can be awkward as well.

Comment (jenf) [reply lib/engine/game/g_18_ireland/misc_config.rb]: discussion complete

**Pull request merged**